### ISSUE TITLE
Using SharePoint Online templates - export

### ISSUE THREAD

**Issue by Cheridbell2** [SharePoint/sp-dev-docs#8303]
Is there anyway to download a Lookbook template as a zip file? The article seems to be missing information.

[Enter feedback here]

Is there anyway to download a Lookbook template as a zip file? The article seems to be missing information.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b9f52c85-8e87-991f-5002-149f011fe8d0
* Version Independent ID: 352955bd-4e1a-126a-a026-440d2b62de38
* Content: [SharePoint site template and site script overview](https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/site-design-overview)
* Content Source: [docs/declarative-customization/site-design-overview.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/declarative-customization/site-design-overview.md)
* Service: **sharepoint-online**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by markgort86** [comment]
I don't think so, please see the documentation at the following URL: https://docs.microsoft.com/en-us/sharepoint/add-sample-site. The only way to provision the templates is to click at the button "Add to your tenant".
